graylog2-web-interface/src/components/lookup-tables/DataAdapterForm.jsx

```diff
@@ -357,7 +363,7 @@ class DataAdapterForm extends React.Component {
               <fieldset>
                 <Row>
                   <Col mdOffset={3} md={9}>
-                    <Button type="submit" bsStyle="success">{create ? 'Create Adapter'
+                    <Button type="submit" bsStyle="success" disabled={!isFormDisabled}>{create ? 'Create Adapter'
```

Comment:
I like the `isFormDisabled` name better as @edmundoa suggested, but right here it's telling the opposite is true. If the form is not not disable, it's disabled. I think the default and this check need to be flipped.

Other than that: looks good to me!

Reply:
Thank you ! fixed it.